### ISSUE TITLE
[Bug]: 'Mod' is case sensitive in keyboard shortcuts

### ISSUE THREAD

**Issue by Nantris** [ueberdosis/tiptap#5399]
### Affected Packages

core

### Version(s)

2.5.4

### Bug Description

I discovered after several hours of hair pulling that `mod` is not the same as `Mod` in keyboard shortcuts. This is exceedingly tedious to work around since ProseMirror already cares about the capitalization of other letters in the string.

### Browser Used

Chrome

### Expected Behavior

`mod-k` should be functionally equivalent to `Mod-k`

### Dependency Updates

- [X] Yes, I've updated all my dependencies.

**Comment by nperez0111** [comment]
Hm I'm unsure what to do about this because we use prosemirror-keymap directly with nothing magic on the tiptap side.

But when I looked into how it works, I think that prosemirror-keymap already does keybind normalization? https://github.com/ProseMirror/prosemirror-keymap/blob/master/src/keymap.ts#L17

And when I tested this myself I was able to have `mod-k` work just fine 🤔 . So I'm unsure about what you are experiencing

**Comment by Nantris** [comment]
I wonder if the `mod` key only has this issue when there's another hotkey, eg `mod+k` versus `Mod+Shift+k`. I'm not really sure. The code you linked (which I already studied while pulling my hair out over this) would certainly lead me to expect this should work under all circumstances, but under at least some circumstances it appears that `Mod` is indeed subject to case sensitivity.

If I uncover more clues, I'll share them here, but I don't expect to since I finally got our hotkey map stabilized.
